docs/manuals/usermans/index.md

```diff
@@ -0,0 +1,84 @@
+# About Taipy's User Manuals
+
+This User Manual covers all the topics and concepts that you can find in Taipy.
+For each topic, we are trying to provide as many examples as possible so that
+you as a Taipy user can perform a specific task.
+
+If you are just starting with Taipy, you may want to look at the
+[Getting Started document](../../getting_started/installation.md)
+in order to see a step-by-step example of a complete Taipy application.
+
+!!! important "Supported Python versions"
+
+    Taipy requires **Python 3.8** or newer.
+
+## Quick Access
+
+These topics are the most visited ones, we thought you’d like to have a glance at them!
+
+<div class="tp-row tp-row--gutter-sm">
+  <div class="tp-col-12 tp-col-md-6 d-flex">
+    <a class="tp-content-card" href="../gui/">
+      <header class="tp-content-card-header">
+        <img class="tp-content-card-icon icon-light" src="../../images/icons/visual-element-w.svg">
+        <img class="tp-content-card-icon icon-dark" src="../../images/icons/visual-element.svg">
+        <h3>Visual elements</h3>
+      </header>
+      <p>
+        Browse dozens of Taipy GUI controls to build your UI
+      </p>
+      <span class="tp-content-card-readmore">Read more</span>
+    </a>
+  </div>
+  <div class="tp-col-12 tp-col-md-6 d-flex">
+    <a class="tp-content-card" href="../gui/viselements/chart/">
+      <header class="tp-content-card-header">
+        <img class="tp-content-card-icon icon-light" src="../../images/icons/bar-chart-w.svg">
+        <img class="tp-content-card-icon icon-dark" src="../../images/icons/bar-chart.svg">
+        <h3>Charts</h3>
+      </header>
+      <p>
+        An important component of Taipy GUI with almost infinite possibilities right at your fingertips.
+      </p>
+      <span class="tp-content-card-readmore">Read more</span>
+    </a>
+  </div>
+</div>
+
+## Graphical User Interface (GUI) - `taipy.gui`
+
+The `taipy.gui^` package allows you to design an effective Graphic User Interface.
+It provides many interactive widgets, controls, and presentative elements to enhance the
+user’s experience.
+
+[:material-arrow-right: GUI User Manual](../gui/index.md)
+
+## Taipy Core - `taipy.core`
+
+The `taipy.core^` package is a Python library designed to build powerful and customized data-driven back-end
+applications.
+It provides the necessary tools to help Python developers transform their algorithms into a complete
+back-end application.
+
+Taipy Core brings algorithm management to another level: algorithms are now connected to the end-user through
+user-defined scenarios, interactive data, smart job orchestration, etc.
+
+
+[:material-arrow-right: Definition of Taipy Core concepts](../core/concepts/index.md),
+
+[:material-arrow-right: Documentation on Taipy Core configuration](../core/config/index.md),
+
+[:material-arrow-right: Description of Taipy Core entities](../core/entities/index.md)
```

Comment:
Let's keep it for now since it is consistent with the actual user-man structure, but I will propose changes once we have designed a new structure.

Reply:
👍